src/components/compress/index.tsx

```diff
@@ -0,0 +1,408 @@
+import { h, Component } from 'preact';
```

Comment:
This is the old `App/index.tsx`

Reply:
anything particularly meaningfully changed here? looks the same at first glance.

Reply:
https://gist.github.com/jakearchibald/7a52541725f34f5f10fff1596febde0d here's a diff